modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliOptions.cs

```diff
@@ -97,7 +97,7 @@ public class LegacyCliOptions : Dictionary<string, string>, IAgentConfigProvider
                     { "op|opcpublishinginterval=", "Default value in milliseconds for the publishing interval " +
                             "setting of the subscriptions against the OPC UA server.",
                         (int i) => this[LegacyCliConfigKeys.OpcPublishingInterval] = TimeSpan.FromMilliseconds(i).ToString() },
-                    { "ct|createsessiontimeout=", "The timeout in seconds used when creating a session to an endpoint.",
+                    { "ct|createsessiontimeout=", "Maximum amount of time that a session should remain open by the OPC server without any activity (session timeout) - to request from the OPC server at session creation.",
```

Comment:
>Maximum amount of time that a session should remain open by the OPC server without any activity (session timeout) - to request from the OPC server at session creation. [](http://example.com/codeflow?start=51&length=167)

can you please split this on 2 or 3 lines so it fits the general code look and feel? Please also specify this value is in seconds

Reply:
done